cmd/checkcommits/checkcommits.go

```diff
@@ -216,6 +232,11 @@ func checkCommitBody(config *CommitConfig, commit *Commit) error {
 		return err
 	}
 
+	if commit.revertCommit {
+		// Don't perform checks on revert commits.
+		return nil
```

Comment:
Will this also ignore the SoB check, @jodh-intel?  If that's the case (and it seems to be from a quick look in a code I'm not familiar with), I think we should do the SoB check before returning nil.

Reply:
Agreed - fixed.